### PR TITLE
[14.0][MIG] stock_available_mrp

### PR DESCRIPTION
**Pull request by florian-dacosta**
Replace https://github.com/OCA/stock-logistics-warehouse/pull/1038

### CODE REVIEW

File: stock_available/models/product_product.py [hunk 1]
```diff
@@ -17,20 +17,13 @@ class ProductProduct(models.Model):
...
-            self._context.get("lot_id"),
-            self._context.get("owner_id"),
-            self._context.get("package_id"),
-            self._context.get("from_date"),
-            self._context.get("to_date"),
-        )
         res = {}
         for product in self:
             res[product.id] = {
-                "immediately_usable_qty": stock_dict[product.id]["virtual_available"],
```
Comment (chienandalu) [inline stock_available/models/product_product.py]: I'd say this was done this way for performance. What's the issue you're trying to solve?

Comment (florian-dacosta) [reply stock_available/models/product_product.py]: I actually made a mistake, my Odoo version was outdated. It is not relevant anymore.
Sorry for the noise, I've update the PR

**Review by chienandalu — APPROVED** [review]
Thanks :slightly_smiling_face: :+1:

**Review by ntsirintanis — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**